embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -86,6 +86,24 @@ public class FlutterApplication : CoreUIApplication, IPluginRegistry
         /// </summary>
         protected bool IsTopLevel { get; set; } = false;
 
+        /// <summary>
+        /// Whether the app should be supported pointing device(mouse) or not.
+        /// If false, pointing device input will not work and and the D-PAD will show with the floating menu.
+        /// If true and IsFloatingMenuSupport is false, pointing device input will not work.
+        /// 
+        /// It only works on TV.
+        /// </summary>
+        protected bool IsPointingDeviceSupport { get; set; } = true;
```

Comment:
I have a question: Is there a place where app developers can search for these APIs?

Reply:
As far as I know, currently, there is no guide and API reference about FlutterApplication(cs, cpp). someday, we will need to make a guide for that APIs  .